nautobot/docs/user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md

```diff
@@ -222,6 +222,10 @@ These endpoints `/ipam/roles/`, `/dcim/rack-roles/` and `/dcim/device-roles/` ar
 | `/dcim/rack-roles/`   | `/extras/roles/` |
 | `/ipam/roles/`        | `/extras/roles/` |
 
+### New Interface to IP Address Relationship Endpoint
```

Comment:
Wonder if it'd be good to document or link to this in `nautobot/docs/user-guide/core-data-model/ipam/ipaddress.md` and/or the interface and vminterface docs as well?